Cargo.toml

```
@@ -1,10 +1,10 @@
 [package]
-name = "nameshed-hsm-relay"
+name = "kmip2pkcs11"
 version = "0.1.0"
 edition = "2024"
-authors = ["NLnet Labs <nameshed@nlnetlabs.nl>"]
-description = "KMIP to PKCS#11 HSM relay for the Nameshed project."
-repository = "https://github.com/NLnetLabs/nameshed-hsm-relay/"
+authors = ["NLnet Labs <rust-team@nlnetlabs.nl>"]
```

going by the aliases file, don't you want?

```suggestion
authors = ["NLnet Labs <dns-rust-team@nlnetlabs.nl>"]
```

---

No, because it's not DNS specific.